routes/v0/post-surveys.js

```diff
@@ -0,0 +1,39 @@
+const models = require('../../models');
+const { SafeRouter, publicEndpoint } = require('../../permissions');
+
+const { PostSurvey } = models;
+const PostSurveyController = require('../../controllers/post-survey-controller')(PostSurvey);
+
+const postSurveysRouter = SafeRouter();
+
+postSurveysRouter.get('/', publicEndpoint, async (
```

Comment:
This should definitely not be public or available with just a key.  You can check with Dee but at a minimum, these GETs should require supervisor privileges.  Also for Line 28.  This is very important.